kolibri/core/assets/src/views/ExamReport/CurrentTryOverview.vue

```diff
@@ -171,7 +175,9 @@
         return this.currentTryDefined && !isUndefined(this.currentTry.correct);
       },
       score() {
-        return this.currentTryDefined ? this.currentTry.correct / this.totalQuestions : 0;
```

Comment:
Have lost the division here in the small refactor.

Reply:
Will update the test scenario to be more robust thank you!